modules/openapi-generator/src/main/resources/elixir/connection.ex.mustache

```diff
@@ -155,7 +155,7 @@ defmodule {{moduleName}}.Connection do
     of the function call, will be set as a bearer token in the
     `authorization` header.
   - `scopes`: a list of Strings represenging OAuth2 scopes.
-  - `options`: a keyword list of OpenAPIPetstore.Connection.options.
+  - `options`: a keyword list of {{moduleName}}.Connection.options.
```

Comment:
looks like a rebase failed and some other changes not authored by you are also included.

given that only a few elixir related changes (already merged into master) are included, we will accept these this time.